Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -65,7 +65,7 @@ struct PitchSpeller {
     // MARK: - Initializers
 
     /// Create a `PitchSpeller` to spell the given `pitches`, with the given `parsimonyPivot`.
-    init(pitches: [Pitch], parsimonyPivot: Pitch.Spelling<EDO12> = .init(.d)) {
+    init(pitches: [Int: Pitch], parsimonyPivot: Pitch.Spelling<EDO12> = .init(.d)) {
```

Comment:
Should the caller need to supply the index here (as part of the API), or should that be done as part of an implementation detail. E.g.,

```Swift
init(pitches: [Pitch], ...) {
    self.pitches = Dictionary(pitches.enumerated())
    ...
}
```

* not verified, but something like that?

Reply:
In a sense, I don't think the user of `Pitch.Speller` should even know why such indices would exist.

Reply:
Ahh, jeez, just started a review by accident.

Reply:
I figure that this `Int` may one day become a more general unique EventID tied to something in the score model, so you would not necessarily have `0,1,2,3,...`, and so I do want to explicitly tie each `Pitch` value to its index.

Reply:
Ahh, interesting.

Perhaps there could be a context-less API for the speller. And then a progressively-disclosed increasingly-contextual set of APIs (e.g., to handle adjacency, segmentation, etc.).

Reply:
Whoops, was that PR description there the whole time 🤦‍♂️

Reply:
I do think the PitchSpeller will need to be passed adjacency information in addition to raw pitches as the same set of pitches can get spelled differently depending on how close they are to each other and in what order they occur (basically adjacency info)

Reply:
I am thinking of the `Int` as the `Node` values of the adjacency graph that get passed in

Reply:
... and which also contribute to the `typealias NewNode = Cross<Node, Tendency>` node values in the `flowNetwork`